code/parachain/frame/composable-support/src/math/safe.rs

```diff
@@ -5,9 +5,11 @@ use sp_runtime::{
 	ArithmeticError,
 };
 
-/// Thin wrapper around [`multiply_by_rational`], transforming any errors into [`ArithmeticError`]
-/// for easier use in pallets and Dispatch-related contexts. See [`multiply_by_rational`] for more
-/// information.
+/// Thin wrapper around [`multiply_by_rational_with_rounding`], transforming any errors into
+/// [`ArithmeticError`] for easier use in pallets and Dispatch-related contexts. See
+/// [`multiply_by_rational_with_rounding`] for more information.
+///
+/// Computes a * b / c, rounding down.
```

Comment:
Do we want to expose the rounding as a parameter?

Reply:
Perhaps, but this was just a documentation update as it had fallen out of sync with the actual function implementation. It can be addressed in the future :+1: